core-rust/state-manager/src/jni/state_manager.rs

```diff
@@ -131,6 +132,7 @@ pub struct StateManagerConfig {
     pub database_backend_config: DatabaseBackendConfig,
     pub database_flags: DatabaseFlags,
     pub logging_config: LoggingConfig,
+    pub no_fees: bool,
```

Comment:
I'm not a massive fan of random booleans, but I guess this is okay for now until we have more configurations we might want to combine into a more meaningful struct.